### ISSUE TITLE
fc-iconset is not compatible with 24.0.0.beta1

### ISSUE THREAD

**Issue by javier-godoy** [FlowingCode/FontAwesomeIronIconset#61]
Using version 4.2.0 of the add-on with Vaadin 24.0.0.beta1 fails with `Uncaught (in promise) TypeError: this.__getIconId is not a function`

https://github.com/FlowingCode/FontAwesomeIronIconset/blob/5d541115e059c88ef939124f9fff2e9828b348f7/src/main/javascript/font-awesome-iron-iconset/fc-iconset.js#L30

**Comment by javier-godoy** [comment]
`__getIconId` was removed in https://github.com/vaadin/web-components/pull/5334

**Comment by javier-godoy** [comment]
In vaadin 24 iconsets are registered without adding to DOM. This change requires a new major version of the add-on.

```diff
-import '@vaadin/icon/vaadin-iconset.js';
+import { Iconset } from '@vaadin/icon/vaadin-iconset.js';

template.innerHTML = `
-<vaadin-iconset name="lumo" size="1000">
<svg xmlns="http://www.w3.org/2000/svg">
<defs>...</defs>
</svg>
-</vaadin-iconset>
`;

-document.head.appendChild(template.content);
+Iconset.register('lumo', 1000, template);
```

**Comment by javier-godoy** [comment]
Will be released in 5.0.0
